packages/vitepress-plugin-git-changelog/src/client/components/Changelog.vue

```diff
@@ -112,7 +122,11 @@ const isFreshChange = computed(() => {
             }) }}
           </span>
         </span>
-        <input v-model="toggleViewMore" type="checkbox" invisible appearance-none>
+        <!-- <input v-model="toggleViewMore" type="checkbox" invisible appearance-none> -->
+        <svg
+          :class="isDescending ? 'i-octicon:sort-desc-16' : 'i-octicon:sort-asc-16'" ml-auto mr-4 cursor-pointer
+          @click.stop="toggleViewMore && (isDescending = !isDescending)"
+        />
```

Comment:
```suggestion
        <div
          :class="isDescending ? 'i-octicon:sort-desc-16' : 'i-octicon:sort-asc-16'" ml-auto mr-4 cursor-pointer
          @click.stop="toggleViewMore && (isDescending = !isDescending)"
        />
```

What was the difference between `svg` and `div` in such scenario?

Reply:
In my opinion, image should be img or svg or other image related html tag rather than span or div, but this is up to you.